Fitness/Views/AllProRoutinesView.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  AllProRoutinesView.swift
+//  Fitness
+//
+//  Created by Cameron Hamidi on 3/21/19.
+//  Copyright © 2019 Keivan Shahida. All rights reserved.
+//
+
+import UIKit
+
+class AllProRoutinesView: UIView {
+
+    let interViewSpacing = 16
```

Comment:
This can just go at the top of the function it's used in if it is only used once.

Reply:
Understood